unime/src-tauri/tests/fixtures/states/shenron_six_credentials_two_connections.json

```diff
@@ -0,0 +1,220 @@
+{
+    "profile_settings":{
+        "locale":"en-US",
```

Comment:
It doesn't seem like this json file is formatted accordingly, since there are spaces missing between they keys and values, such as `"locale":"en-US"`.

Reply:
well npm run format and cargo fmt didn't fix it, but i've changed all ":" to ": "
Was there anything else formatted incorrectly that I missed?

Reply:
`npm run format` is currently configured to ignore the entire `src-tauri/*` folder (not to interfere with `cargo fmt`). I will try to set up an exception for json files.